index.js

```diff
@@ -1034,22 +1099,19 @@ const fileType = input => {
 			mime: 'application/x-esri-shape'
 		};
 	}
-};
-
-module.exports = fileType;
-
-Object.defineProperty(fileType, 'minimumBytes', {value: 4100});
+}
 
-fileType.stream = readableStream => new Promise((resolve, reject) => {
+const stream = readableStream => new Promise((resolve, reject) => {
 	// Using `eval` to work around issues when bundling with Webpack
 	const stream = eval('require')('stream'); // eslint-disable-line no-eval
 
 	readableStream.on('error', reject);
-	readableStream.once('readable', () => {
+	readableStream.once('readable', async () => {
```

Comment:
You cannot just mark this as `async`. If anything throws it will cause an uncaught rejection. You need to wrap the whole body in a try/catch and then properly `reject` in the catch.

Reply:
In either code, in case of reject, resolve will be called as well.,,